### DISCUSSION TITLE
Resources needed?

### DISCUSSION THREAD

**Discussion by Ciao121** [bluenviron/mediamtx#3469] [Questions]
### Question

I know that resources (ram, cpu) needs cannot be the same for all projects but how do I get an idea for a small project? Let say just proxy 5 streams... I cannot find any info about needed resources.

**Comment by moenodedev** [comment]
You can stream hundreds of cameras from a raspberry pi.

**Comment by moenodedev** [comment]
> how do I get an idea for a small project

MediaMTX is easy to run from a laptop

**Comment by aler9** [comment]
Hello, 5 streams can be handled by any kind of hardware, even by a 20$ Raspberry Pi.

When dealing with video streaming, the bottleneck is almost always the bandwidth, not the RAM or the CPU.

If you want to obtain the maximum capability of the system, you have to run tests on the system itself, for instance by increasing the number of proxied sources until warnings start to appear in the console.

**Comment by ghost** [comment]
When you say 5 streams can be handled by a Rasberry Pi, does that include having multiple viewers accessing them via WebRTC?

I have MediaMTX running in a Windows Azure VM, and with 8 streams + 8 users my CPU is around 50%

[a comment by an automated account is not shown]
